Sources/tart/Platform/Platform.swift

```diff
@@ -5,4 +5,5 @@ protocol Platform: Codable {
     func bootLoader(nvramURL: URL) throws -> VZBootLoader
     func platform(nvramURL: URL) -> VZPlatformConfiguration
     func graphicsDevice(vmConfig: VMConfig) -> VZGraphicsDeviceConfiguration
+    func pointingDevices(vmConfig: VMConfig) -> [VZPointingDeviceConfiguration]
```

Comment:
You can probably can skip the `vmConfig` argument here as it's unused.

Reply:
Good catch! See [c1d7ebb](https://github.com/cirruslabs/tart/pull/202/commits/c1d7ebb5ffaa52230d5a37c393e51cbe43d3ee18).